terra-jupyter-python/requirements.txt

```diff
@@ -44,3 +44,5 @@ scikit-learn-intelex
 xgboost
 #older version of nbconvert fail to convert notebooks to html
 nbconvert>=7.7.3
```

Comment:
Interesting that this did not need to change in order to fix the notebook download issue on Terra 🤔

Reply:
hmm that is odd, I poked around a bit and there's an nbconvert_support section of [jupyter_contrib_nbextensions](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master), maybe the old version of that was incompatible for some reason